package.json

```diff
@@ -19,10 +19,10 @@
     "build:dev": "webpack --config webpack/webpack.config.js --env.NODE_ENV=dev",
     "build:prev": "webpack --config webpack/webpack.config.js --env.NODE_ENV=prev",
     "build:prod": "webpack --config webpack/webpack.config.js --env.NODE_ENV=prod",
-    "bundleStats:dev": "webpack --env.NODE_ENV=dev --profile --json > stats.json",
-    "bundleStats:dev:windows": "webpack --env.NODE_ENV=dev --profile --json | Out-file 'stats.json' -Encoding OEM",
-    "bundleStats:prod": "webpack --env.NODE_ENV=prod --profile --json > stats.json",
-    "bundleStats:prod:windows": "webpack --env.NODE_ENV=prod --profile --json | Out-file 'stats.json' -Encoding OEM",
+    "bundleStats:dev": "webpack --config webpack/webpack.config.js --env.NODE_ENV=dev --profile --json > stats.json",
```

Comment:
should have made this change when webpack configs were moved to `<root>/webpack`